test/maxcdn_test.rb

```diff
@@ -8,6 +8,7 @@
 
 require "webmock"
 include WebMock::API
+WebMock.enable!
```

Comment:
Update to support newest version of webmock. This is why the first travis tests failed.